config.py

```diff
@@ -64,6 +64,9 @@ class TestingConfig(BaseConfig):
     DEBUG = True
     TESTING = True
 
+    # mail accounts
+    MAIL_DEFAULT_SENDER = 'certain@example.com'
```

Comment:
why do you need this email here?

Reply:
It is used by mail api while sending messages. This email was already used in dev environment and we also definitely need it in test environment!

Reply:
Ah ok, is just for the testing config. That's ok then :)